app/models/payment.rb

```diff
@@ -0,0 +1,45 @@
+class Payment < ActiveRecord::Base
+  has_many :ticket_purchases
+  belongs_to :user
+  belongs_to :conference
+
+  attr_accessor :stripe_customer_email
+  attr_accessor :stripe_customer_token
+
+  validates :status, presence: true
+  validates :amount, presence: true, numericality: { greater_than: 0 }
+  validates :user_id, presence: true
+  validates :conference_id, presence: true
+
+  enum status: {
+    unpaid: 0,
+    success: 1,
+    failure: 2
+  }
+
+  def amount_to_pay
+    Ticket.total_price(conference, user, paid: false).cents
+  end
+
+  def purchase
+    customer = Stripe::Customer.create email: stripe_customer_email,
+                                       source: stripe_customer_token,
+                                       description: user.name
```

Comment:
Actually as we make the payment instantly, I think we can call Stripe's `Charge#create` directly instead of creating and saving customer details in our DB. 
I will remove new Customer creation part.